src/index.js

```diff
@@ -226,10 +255,25 @@ const HMMediaWeightSidebar = () => {
 											}
 										</strong>
 									</p>
+
 									<p>
 										Attachment ID: { attachment.id }<br />
 										<small><a href={ `upload.php?item=${ attachment.id }` }>Go to the attachment post &rsaquo;</a></small>
 									</p>
+
+									{ requestedSize === 'full' && (
+										<>
+											<Flex direction="row" align="flex-start" gap={ 6 }>
+												<FlexItem>
+													<Icon icon={ caution } size={ 36 } />
+												</FlexItem>
+												<FlexItem>
+													<p><strong>{ __( 'Full size image requested. Edit block to request smaller image.', 'hm-media-weight' ) }</strong></p>
```

Comment:
I don't think this warning makes sense. This isn't necessarily requesting the full-size image; it's requesting an uncropped image, which in theory should be output at the appropriate resolution for the browser.